AjaxControlToolkit/Rating/Rating.cs

```diff
@@ -401,13 +395,25 @@ public class Rating : Panel, ICallbackEventHandler, IPostBackEventHandler {
         /// </summary>
         /// <param name="eventArgument" type="String">Event argument</param>
         public void RaisePostBackEvent(string eventArgument) {
-            var args = new RatingEventArgs(eventArgument);
+            HandleEvent(eventArgument);
+        }
+
+        private RatingEventArgs HandleEvent(string eventArgument) {
+            var eventArgumentParts = eventArgument.Split(';');
+            var eventName = eventArgumentParts[0];
 
-            OnClick(args);
+            var args = new RatingEventArgs(eventArgumentParts[1] + ";" + eventArgumentParts[2]);
```

Comment:
Since we already do split, we can change the `RatingEventArgs` to accept 2 arguments